examples/catalog-management.v1.test.js

```diff
@@ -2,7 +2,7 @@
  * @jest-environment node
```

Comment:
This file is missing example code blocks for deleteShareApprovalList() and getOfferingSourceArchive()